### DISCUSSION TITLE
Support for Apple Mac M1?

### DISCUSSION THREAD

**Discussion by HeyWilliam99** [QuSpin/QuSpin#475] [General]
Does the QuSpin package support the Apple Mac M1 chip?  Had anyone ever tried?

**Comment by mgbukov** [comment]
I don't know of any tests yet; why would you expect python to behave any different on Mac M1 than on other macs?

**Reply by HeyWilliam99** [reply]
Because the framework of M1 silicon is ARM which is quite different from Intel X86, many package need new versions to support M1 silicon like Numpy. Hope someone do a test, so I can decide whether to purchase Macs with m1 for computing.

**Reply by weinbe58** [reply]
I am aware of the new ARM architecture for Macs. The issue is if NumPy is not supported yet we devinitely can't because we are heavily dependent on NumPy.

**Reply by j1ver** [reply]
I compiled it myself, even with OpenMp support. It works just fine :) for installing the requiered packages I use miniforge

**Reply by dingchengxiang2022** [reply]
> I compiled it myself, even with OpenMp support. It works just fine :) for installing the requiered packages I use miniforge

how to do that? could you show more details?
